README.md

```diff
@@ -127,6 +127,109 @@ The following options are available:
 : A SQL statement that is executed before inserting the data, e.g. to create the target table.
 : *NOTE:* the underlying database should support the *create table if not exist* idiom, as the plugin will execute this statement every time the script is run.
 
+## SQL Execution Functions
+
+This plugin provides the following functions for executing SQL statements that don't return data, such as DDL (Data Definition Language) and DML (Data Manipulation Language) operations.
+
+### execute
+
+The `execute` function executes a SQL statement that doesn't return a result set, such as `CREATE`, `ALTER`, `DROP`, `INSERT`, `UPDATE`, or `DELETE` statements. For example:
+
+```nextflow
+include { execute } from 'plugin/nf-sqldb'
+
+// Create a table
+execute(
```

Comment:
How errors are managed? is it trowing an exception? maybe using a result value is a bit simpler for a nextflow user instead of using try/catch statements